package.json

```diff
@@ -25,9 +25,9 @@
   },
   "devDependencies": {
     "@acme/prettier-config": "workspace:*",
-    "@turbo/gen": "^2.1.3",
+    "@turbo/gen": "~2.1.3",
     "prettier": "catalog:",
-    "turbo": "^2.1.3",
+    "turbo": "~2.1.3",
```

Comment:
??

Reply:
Turbo 2.2 crashes my projects in my work repo. I think it also did in this one. Not sure. I'll try again

Reply:
Yup. Also panics here

Reply:
Yeah, it happens with me as well. 2.2.3 is a bit better, but it still happens